### ISSUE TITLE
Nail elevation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10710]
**Preferred term label:**
Nail elevation

**Synonyms**

**Definition (free text, please give PubMed ID)**
Elevation of the nail from the nail bed

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Seymour fracture
Subungual nevi
Digital mucous cyst
Phalanx fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as 
HP:0025801
Nail elevation
